safe-redirect-manager.php

```diff
@@ -28,5 +77,5 @@
 	WP_CLI::add_command( 'safe-redirect-manager', 'SRM_WP_CLI' );
```

Comment:
```suggestion
	\WP_CLI::add_command( 'safe-redirect-manager', 'SRM_WP_CLI' );
```
Or maybe better to prefix added functions with `srm_`? As we are not using `namespace` throughout the plugin. 

Thanks